crates/sel4-capdl-initializer/core/src/lib.rs

```diff
@@ -359,6 +359,28 @@ impl<'a, N: ObjectName, D: Content, M: GetEmbeddedFrame, B: BorrowMut<[PerObject
                                 }
                             }
                         }
+                        #[sel4_cfg(any(ARCH_IA32, ARCH_X86_64))]
```

Comment:
Would you mind replacing this condition with just `sel4_cfg(ARCH_X86_64)`? I've been trying to avoid mention of IA32 to prevent confusion.